### PR TITLE
Event page refactor

### PR DESCRIPTION
**Pull request by ghost**
### event-item
- cleaned up css
- if-else block used for displaying event location status
- scale animation on hover
- removed colored borders from event box to match design guidelines
- join btn works visually like other site buttons now

### common-hero
- mobile breakpoint to define margins
- new width

### acm-toaster
- removed colored borders to match design guidelines
- added brand-med to toastItem.content

### about/index
- modified spacing components
- lowercased E in event to match headers
- fixed text sizing

### CODE REVIEW

**Comment by jaasonw** [comment]
image

also kinda not liking the contrast here if ima be honest

**Comment by jaasonw** [comment]
can you animate the opening and closing of this too
see: https://svelte.dev/repl/74ff0da7cb074d4788b996e2d91258d3?version=3.23.0

File: src/lib/components/events/event-item.svelte [hunk 1]
```diff
@@ -39,17 +38,20 @@
...
       <div class="event-name">
         <h2 class="headers">
           {info.title}
         </h2>
 
-        <p class="event-location">
-          {info.location === 'Discord' || info.location === 'Zoom'
-            ? `Hosted on ${info.location}`
-            : info.location}
+        <p class="event-location brand-med acm-gray">
```
Comment (jaasonw) [inline src/lib/components/events/event-item.svelte:50]: Is discord what gets inputted to indicate default? What happens with the case where it actually is hosted on discord?

File: src/lib/components/events/event-item.svelte [hunk 2]
```diff
@@ -58,12 +60,12 @@
         <time>
           {info.month}
           {info.day} at {info.time}
-          {#if isRecurring}(recurring){/if}
+          {#if isRecurring}<br />Recurring{/if}
```
Comment (jaasonw) [inline src/lib/components/events/event-item.svelte:64]: image

Is there a way to verify that this works? The only recurring event doesn't actually indicate recurring

File: src/lib/components/utils/acm-toaster.svelte [hunk 3]
```diff
@@ -58,7 +57,7 @@
     align-self: center;
   }
 
-  @media (min-width: 800px) {
+  @media (min-width: 900px) {
```
Comment (jaasonw) [inline src/lib/components/utils/acm-toaster.svelte:60]: can you also fix #408 with this

File: src/routes/events/index.svelte [hunk 4]
```diff
@@ -37,15 +35,15 @@
 
 <Spacing --min="100px" --med="175px" --max="200px" />
 
-<h2 class="size-lg headers">Upcoming Events 📅</h2>
+<h2 class="size-lg headers">Upcoming events 📅</h2>
```
Comment (jaasonw) [inline src/routes/events/index.svelte:39]: #445

**Review by jaasonw — CHANGES_REQUESTED** [review]

File: src/lib/components/events/event-item.svelte (hunk 1 above)
Comment (jaasonw) [reply src/lib/components/events/event-item.svelte:50]: i clicked on the wrong line i meant to comment on the next one

File: src/routes/events/index.svelte (hunk 4 above)
Comment (jaasonw) [reply src/routes/events/index.svelte:39]: how r u just gonna revert @karnikaavelumani only pull request like that

Comment (k4rni) [reply src/routes/events/index.svelte:39]: BRUH FR LOOK AT WHAT HE DID @Peekoe ALL OUR EFFORT

Comment (charliettaylor) [reply src/routes/events/index.svelte:39]: How dare they remove the

**Comment by ghost** [comment]
> 

That was on purpose

**Comment by ghost** [comment]
I PRESSED THE WRONG BUTTON

File: src/routes/events/index.svelte (hunk 4 above)
Comment (ghost) [reply src/routes/events/index.svelte:39]: It doesn't follow how the rest of our headers are

File: src/lib/components/utils/acm-toaster.svelte (hunk 3 above)
Comment (ghost) [reply src/lib/components/utils/acm-toaster.svelte:60]: Toasts centered in landscape during my testing. will double check tho

**Comment by ghost** [comment]
> image
> 
> also kinda not liking the contrast here if ima be honest

I dont either lol still trying to find a better solution

File: src/lib/components/events/event-item.svelte (hunk 2 above)
Comment (ghost) [reply src/lib/components/events/event-item.svelte:64]: that's weird bc i remember it did indicate recurring. will check

File: src/lib/components/events/event-item.svelte (hunk 1 above)
Comment (ghost) [reply src/lib/components/events/event-item.svelte:50]: Discord is the default. Janky work around would be to set the location on GCal for an event to be 'Discord' if we are hosting on discord, but i'm willing to see if i can add something to make an exception

**Comment by ghost** [comment]
> 
> 
> can you animate the opening and closing of this too see: https://svelte.dev/repl/74ff0da7cb074d4788b996e2d91258d3?version=3.23.0

oh yeah lol will do

File: src/lib/components/utils/acm-toaster.svelte (hunk 3 above)
Comment (jaasonw) [reply src/lib/components/utils/acm-toaster.svelte:60]: referring to the fact that all the toasts take the width of the widest existing toast so it snaps back when it disappears

File: src/routes/events/index.svelte (hunk 4 above)
Comment (jaasonw) [reply src/routes/events/index.svelte:38]: counterproposal: make the rest of the headers match

Comment (jaasonw) [reply src/routes/events/index.svelte:39]: counterproposal: make the rest of the headers match

Comment (k4rni) [reply src/routes/events/index.svelte:39]: @jaasonw check discord poll on issue

File: src/lib/components/events/event-item.svelte [hunk 5]
```diff
@@ -6,7 +6,6 @@
   import CopyTextIcon from '$lib/components/icons/copy-text.svelte';
   import GoogleCalendarIcon from '$lib/components/icons/google-calendar.svelte';
   import MsOutlookIcon from '$lib/components/icons/ms-outlook.svelte';
-
```
Comment (EthanThatOneKid) [inline src/lib/components/events/event-item.svelte:9]: Nit: Why was this line removed?

File: src/routes/events/index.svelte [hunk 6]
```diff
@@ -1,31 +1,29 @@
 <script lang="ts" context="module">
   import type { LoadInput, LoadOutput } from '@sveltejs/kit/types/internal';
-
```
Comment (EthanThatOneKid) [inline src/routes/events/index.svelte]: Nit: Why was this line removed?

File: src/routes/events/index.svelte [hunk 7]
```diff
@@ -57,7 +55,6 @@
     justify-content: center;
     text-align: center;
   }
-
```
Comment (EthanThatOneKid) [inline src/routes/events/index.svelte:60]: Nit: Why was this line removed?

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
Huge changes! But why did you remove these lines?

File: src/lib/components/utils/acm-toaster.svelte (hunk 3 above)
Comment (ghost) [reply src/lib/components/utils/acm-toaster.svelte:60]: can u send video demo

File: src/lib/components/events/event-item.svelte (hunk 5 above)
Comment (ghost) [reply src/lib/components/events/event-item.svelte:9]: I think that was lint

File: src/routes/events/index.svelte (hunk 6 above)
Comment (ghost) [reply src/routes/events/index.svelte]: I think that was lint

File: src/routes/events/index.svelte (hunk 7 above)
Comment (ghost) [reply src/routes/events/index.svelte:60]: Lint
